From d16afd2cb65f6cc1e0522a11e7d5a50e5cfe9e36 Mon Sep 17 00:00:00 2001
From: Theodore Ts'o <tytso@mit.edu>
Date: Wed, 22 Aug 2018 16:33:10 -0400
Subject: [PATCH] debian: remove hard-coded dependency for libss2 in
 e2fsprogs.shlibs.local

The hard-coded override is very old, and isn't necessary --- in fact,
it's actively harmful because we are using one or two symbols that
were added to libss2 after e2fsprogs 1.34.   So remove it.

Signed-off-by: Theodore Ts'o <tytso@mit.edu>
---
 debian/e2fsprogs.shlibs.local | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/e2fsprogs.shlibs.local b/debian/e2fsprogs.shlibs.local
index 3ef2b64b3..a99d2a0a8 100644
--- a/debian/e2fsprogs.shlibs.local
+++ b/debian/e2fsprogs.shlibs.local
@@ -1,3 +1,2 @@
-libss 2 libss2 (>= 1.34-1)
 libext2fs 2 libext2fs2 (= ${binary:Version})
 libe2p 2 libext2fs2 (= ${binary:Version})
